frontend/components/footer/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import './footer.css';
+
+const Footer = ({ data }) => {
```

Comment:
check the passed data, to render the component only if it passed the check

Reply:
since this is next and not react it renders the whole page not by components so the data check is not needed on the component level

Reply:
the data check is highly required on the component level as we fetch its data from the cms and in some cases the component data is not found ( or created)  so we don't want the component to throw an error on our face 😄

Reply:
ok so this is the only issue you have with this component ? i tried to fix the other issues if this is the only issue i'll tell how we fix it later today :]